packages/core/src/app/checkout/CheckoutStep.spec.tsx

```diff
@@ -46,7 +46,11 @@ describe('CheckoutStep', () => {
     });
 
     afterEach(() => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        //@ts-ignore
         delete window.scrollTo;
```

Comment:
It appears a lot of `@ts-ignore` around `window` object. What would be the cost of avoiding `@ts-ignore`?